package.json

```diff
@@ -261,11 +261,11 @@
     },
     {
       "path": "./dist/antd.min.css",
-      "maxSize": "55 kB"
+      "maxSize": "70 kB"
```

Comment:
Should use latest config

Reply:
my rtl classes increased final css size. so if i decrease `maxSize` ci will fail

Reply:
👌